main.js

```diff
@@ -1,96 +1,137 @@
+    playerScore.classList.add("changingScore");
+
   } else {
-    computerWins = computerWins+1; 
-    return "Computer";
+    const computerScore = document.getElementById("scoreboard-ComputerScore");
+    computerScore.addEventListener("animationend", () => {
+      computerScore.textContent = computerWins;
+      computerScore.classList.remove("changingScore");
+    });
+    computerScore.classList.add("changingScore");
+    computerWins = computerWins + 1;
   }
-}
+  
+  const computerButtons = document.querySelector(".choicesC .buttons");
+  const computerSelection = computerButtons.querySelector(
+    `[value="${choiceC}"]`
+  );
+  computerSelection.addEventListener("animationend", () => {
+    computerSelection.classList.remove("chosen");
+  });
+  computerSelection.classList.add("chosen");
 
-function logWins() {
-  const output = `Player won ${playerWins} times, Computer won ${computerWins} times, draws happened ${draws} times`
-  alert(output);
-  alert(getChampionOutput());
 }
+let popup = document.getElementById("popup");
 
-function logRound(playerChoice, computerChoice, winner, roundNumber) {
-  const output = `Round ${roundNumber+1}: you selected ${playerChoice}, computer selected ${computerChoice}, winner is ${winner}`
-  alert(output)
-}
+function confirmGameEnd() {
+  const overlay = document.getElementById("overlay");
 
-function getChampionOutput() {
-  if (playerWins > computerWins) {
-    return "You are adopted and a winner";
-  } else if (computerWins > playerWins) {
-    return "You are adopted and can't beat a computer";
-  } else {
-    return "go commit unalive";
+  if (playerWins == 5) {
+    overlay.classList.add("active");
+    popup.style.display = "flex";
+    popup.classList.add("active");
+  
+    playerWinsBanner.removeAttribute("hidden");
+    restartBtnPlayer.removeAttribute("hidden");
+    rockBtn.disabled = true;
+    paperBtn.disabled = true;
+    scissorsBtn.disabled = true;
+    resetVariables();
+  } else if (computerWins == 5) {
+    overlay.classList.add("active");
+    popup.style.display = "flex";
+    popup.classList.add("active");
+
+    computerWinsBanner.removeAttribute("hidden");
+    restartBtnComputer.removeAttribute("hidden");
+    rockBtn.disabled = true;
+    paperBtn.disabled = true;
+    scissorsBtn.disabled = true;
+    resetVariables();
   }
 }
 
-function resetValues(){
-  draws = 0;
+
+
+
+function resetVariables() {
   playerWins = 0;
   computerWins = 0;
+  draws = 0;
+}
+
+function restartGame() {
+  let restartBtnComputer = document.getElementById("restart-btnComputer");
+  const playerScore = document.getElementById("scoreboard-PlayerScore");
+  playerScore.innerHTML = playerWins;
+  const computerScore = document.getElementById("scoreboard-ComputerScore");
+  computerScore.innerHTML = computerWins;
+
+  document.getElementById("rock-btn").disabled = false;
+  document.getElementById("paper-btn").disabled = false;
+  document.getElementById("scissors-btn").disabled = false;
```

Comment:
Thos guys are already declared on top of the file. Use `rockBtn.disabled = false` ....

Reply:
```suggestion
  rockBtn.disabled = false;
  paperBtn.disabled = false;
  scissorsBtn.disabled = false;
```